model-optimizer/extensions/middle/ReverseV2ToReverseSequence.py

```diff
@@ -32,29 +35,51 @@ def pattern():
 
     def replace_pattern(self, graph: Graph, match: dict):
         reverse = match['reverse']
```

Comment:
please use `for reverse in graph.get_op_nodes(op='Reverse'):` instead of pattern